oauth-server/src/Dockerfile

```diff
@@ -14,4 +14,5 @@ WORKDIR /app
 EXPOSE 8080
 ENV ASPNETCORE_URLS=http://*:8080
 COPY --from=build /app/publish .
+RUN chmod a+rwx -R ./Data
```

Comment:
There may be a better way of doing this.  I'll merge for now but we should discuss at some point.